CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 85.5.1
+
+This patch release makes a FlexibleHeader dependency on MaterialMath explicit in MaterialComponents.podspec so that pod lib lint validations pass and the release can be published successfully with Cocoapods. There are no code changes.
```

Comment:
can we add the PR/commit to the changelog, or is it not generated as it doesn't belong to any component?

Reply:
As a hotfix, the change list has to be generated manually.

Reply:
I think it's `scripts/release notes`

Reply:
It wasn't initially automatically generated for me, but then again, my apidiff is broken, so maybe this is something that's broken too?

Running `scripts/release notes` just now yielded this:
```
## Multi-component changes

* [Bumped version number to 85.5.1.](https://github.com/material-components/material-components-ios/commit/930e75411215cdd8c594c268753ac4ea8b0f805a) (Andrew Overton)
```

Should I add it?

Reply:
Hotfix releases never get the notes initially. Once you cherry-pick the commit, re-running the appropriate command (it may not be `notes`) will give you the logs to paste into CHANGELOG.md

Reply:
I guess you can just write it by hand. I can't get it to generate, either.

Reply:
Yeah, I think it's `notes`. https://github.com/material-components/material-components-ios/blob/develop/contributing/releasing.md#consider-running-scriptsrelease-notes-again.

Should the output include the "bump" commit? :/

Reply:
No, it generally does not.